plugins/com.google.cloud.tools.eclipse.appengine.libraries.ui/src/com/google/cloud/tools/eclipse/appengine/libraries/ui/LibrarySelectorGroup.java

```diff
@@ -89,8 +89,15 @@ private void createContents(Composite parentContainer, String groupLabel) {
     for (Library library : availableLibraries.values()) {
       Button libraryButton = new Button(apiGroup, SWT.CHECK);
       libraryButton.setText(getLibraryName(library));
-      if (library.getToolTip() != null) {
-        libraryButton.setToolTipText(library.getToolTip());
+      String toolTip = library.getToolTip();
+      
+      String stage = library.getLaunchStage();
+      if (!"GA".equals(stage)) { //$NON-NLS-1
```

Comment:
The code previous guarded against the tooltip being `null`.

Reply:
done